lighthouse-core/test/report/report-generator-test.js

```diff
@@ -117,6 +118,15 @@ describe('ReportGenerator', () => {
       }
     });
 
+    it('creates CSV for results including overall category scores', async () => {
+      const csvOutput = ReportGenerator.generateReport(sampleResults, 'csv');
+      csvOutput.includes('performance-score');
+      csvOutput.includes('accessibility-score');
+      csvOutput.includes('best-practices-score');
+      csvOutput.includes('seo-score');
+      csvOutput.includes('pwa-score');
```

Comment:
```suggestion
      expect(csvOutput).toContain('performance-score');
      expect(csvOutput).toContain('accessibility-score');
      expect(csvOutput).toContain('best-practices-score');
      expect(csvOutput).toContain('seo-score');
      expect(csvOutput).toContain('pwa-score');
```

Reply:
currently it's not actually asserting :)

Reply:
Oh yeah i missed that